### PR TITLE
Dependency removed

### PR DESCRIPTION
**Pull request by codingblazer**
#302 
I have made the following changes -

1.Removed dependency 'com.github.paolorotolo:appintro:1.4.0'.
2.Removed files CircleIntro.java , FirstSlide, SecondSlide , ThirdSlide , FourthSlide.java and added CircleIntroActivity (Intro using Viewpager)
 
UI is not changed (checked on multiple devices).

### CODE REVIEW

File: app/src/main/java/com/peacecorps/pcsa/circle_of_trust/slides/CircleIntroActivity.java [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+        }
+
+        @Override
+        public void onPageScrollStateChanged(int arg0) {
+
+        }
+    };
+
+    public class MyViewPagerAdapter extends PagerAdapter {
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/circle_of_trust/slides/CircleIntroActivity.java]: Do not use names like this for classes.

**Review by sandarumk — CHANGES_REQUESTED** [review]
Please add meaning full name for variables. The rest seem to be fine.

File: app/src/main/java/com/peacecorps/pcsa/circle_of_trust/slides/CircleIntroActivity.java (hunk 1 above)
Comment (codingblazer) [reply app/src/main/java/com/peacecorps/pcsa/circle_of_trust/slides/CircleIntroActivity.java]: @sandarumk Thanks for review. I will make the requested changes.

**Comment by codingblazer** [comment]
@sandarumk I have changed the variable names and made comments wherever necessary.

**Comment by sandarumk** [comment]
@codingblazer Please add a meaningful commit messages.

**Comment by codingblazer** [comment]
@sandarumk Sorry about that. I have changed the commit message.

File: app/src/main/java/com/peacecorps/pcsa/circle_of_trust/slides/CircleIntroActivity.java [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        });
+        btnDone.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                loadMainActivity();
+            }
+        });
+    }
+
+    /**
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/circle_of_trust/slides/CircleIntroActivity.java]: Follow the method commenting format. You can refer the Wiki if you need the format.

File: app/src/main/java/com/peacecorps/pcsa/circle_of_trust/slides/CircleIntroActivity.java [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.ImageButton;
+
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.peacecorps.pcsa.R;
+
+public class CircleIntroActivity extends AppCompatActivity {
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/circle_of_trust/slides/CircleIntroActivity.java]: Follow the class commenting format to add the header comments. Refer the wiki if you need the format

**Comment by sandarumk** [comment]
Please follow the guidelines of the repository when you submit a PR. You can find them in the Wiki. The links to the specific articles are in the READ ME file.

**Comment by codingblazer** [comment]
@sandarumk Thanks for reference. I have read all the rules now and will adhere to them in future. Do you want me to change comments in this PR or anything else also.

**Comment by sandarumk** [comment]
Yes I need to change the comments in this PR.

**Review by sandarumk — CHANGES_REQUESTED** [review]

**Comment by codingblazer** [comment]
@sandarumk I have added the class and method comments.

**Comment by codingblazer** [comment]
@sandarumk I just noticed that there are two "Done" on the following screen and both of them were clickable before this PR. I didn't noticed it that time because I assumed "Done" (TextView) is just for indication. I am very sorry for that.
Should I commit implementation of this "Done" (TextView) to this PR only or create separate one.

**Comment by sandarumk** [comment]
Can you open another issue stating the same. Then let's discuss there separately.

**Pull request merged**